dashboard/cron.py

```diff
@@ -235,20 +235,22 @@ def update_canvas_resource(self):
                 status += f"Row {row.id} removed as it is not available\n"
         return status
 
-    # update RESOURCE_ACCESS records from BigQuery
-    def update_with_bq_access(self):
+    # update RESOURCE_ACCESS records from BigQuery or LRS data sources
+    def update_resource_access(self):
+        is_bigquery = settings.DATABASES['LRS']['ENGINE'] == 'google.cloud.bigquery'
```

Comment:
@andrew-gardener I got the follow error when the LRS->ENGINE is set to 'google.cloud.bigquery':

student_dashboard  | django.core.exceptions.ImproperlyConfigured: 'google.cloud.bigquery' isn't an available database backend.
student_dashboard  | Try using 'django.db.backends.XXX', where XXX is one of:
student_dashboard  |     'mysql', 'oracle', 'postgresql', 'sqlite3'

I suggest to use another bool setting param "LRS_BIGQUERY", outside of the LRS params. So for Unizin schools, we do not need to set LRS params at all, with LRS_BIGQUERY=true

Reply:
I can instead add the LRS optionally to the list of databases (if engine != `google.cloud.bigquery`) 

Just trying to get rid of the variables as per https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1210#discussion_r563967258

Reply:
Also sorry about not catching the error

Reply:
That should work, like in settings.py to append LRS variable only when the engine is not of BigQuery:

if LRS_ENGINE != 'google.cloud.bigquery':
    DATABASES.append(
        {
            'LRS': {
                'ENGINE': ENV.get('LRS_ENGINE', ''),
                'NAME': ENV.get('LRS_DATABASE', ''),
                'USER': ENV.get('LRS_USER', ''),
                'PASSWORD': ENV.get('LRS_PASSWORD', ''),
                'HOST': ENV.get('LRS_HOST', ''),
                'PORT': ENV.get('LRS_PORT', 5432),
                'OPTIONS': ENV.get('LRS_OPTIONS', {}),
            }
        }
    )

Could you please update this pull request, so that I can run it again?

Reply:
Your fixes worked!